pinot-common/src/main/java/org/apache/pinot/common/request/AggregationInfo.java

```diff
@@ -24,29 +24,40 @@
  */
```

Comment:
A lot of files below have just formatting changes. Would be great to exclude them from the PR - the changes also seem to be undoing some of the imports organization that was done in a recent PR.

Reply:
thats coming from thrift auto-gen files, not sure how to control them. See the other PR to remove the auto-gen files from source.